src/simulator/simulator.cpp

```diff
@@ -102,12 +110,17 @@ Simulator::Simulator(const std::string& sim_config) :
     vel_(0, {0,0}),
     cur_loc_(0, {0,0}),
     laser_noise_(0, 1),
-    robot_type_(CONFIG_robot_type) {
+    robot_number_(CONFIG_robot_number),
+    topic_prefixs_(CONFIG_topic_prefixs),
+    robot_types_(CONFIG_robot_types){
   truePoseMsg.header.seq = 0;
   truePoseMsg.header.frame_id = "map";
+  // TODO: add static cast robot_type_(static_cast<RobotType>(CONFIG_robot_type))
+  // initialize object lines associated with each robot
```

Comment:
add name in TODO

Reply:
Sorry that TODO is no longer necessary after the change from enumerated int to string for RobotType. I have removed the comments